controllers/storagecluster/cephcluster.go

```diff
@@ -708,6 +708,10 @@ func getMonCount(nodeCount int, arbiter bool) int {
 		}
 	}
 
+	if monCount != 0 {
+		return monCount
+	}
```

Comment:
what is the behavior, if arbiter is enabled and someone is setting the MonCount to `3` as well?

if we always want to stay at `5` for arbiter move this check to the end of the function.

Reply:
Right, we would want to stay at 5 if arbiter mode is enabled.
Sorry for the confusion here :/